warehouse/static/sass/warehouse.scss

```diff
@@ -43,7 +43,7 @@
 
 // RESETS LAYER: applies CSS resets and sensible defaults
 @import "resets/boxsizing";
-@import "resets/normalize";
+@import "~normalize.css/normalize";
```

Comment:
What does ~ prefix mean?

Reply:
Something I saw meaning `node_modules/normalize.css` but I have absolutely no idea whether it will work. I've been struggling with docker issues recently, so I have yet to test this.

Reply:
Have you seen https://github.com/pypa/warehouse/pull/10052? I may need to squash it, but it ensures that `docker-compose` setup works okay on a clean system.

Reply:
Yeah, my problem was not with the image but with docker itself. See https://github.com/docker/compose/issues?q=is%3Aissue+sort%3Aupdated-desc+author%3Aewjoachim+ for details :)

Reply:
Aaaaaand the tilde is gone.

Reply:
https://github.com/pypa/warehouse/pull/9993 fixes your problem with running this with `docker-compose` by using standard Postrgres import mechanism that supports .xz files without `xz` dependency and pipes.
```
xz -d -f -k dev/example.sql.xz --stdout | docker-compose run --rm web psql -h db -d warehouse -U postgres -v ON_ERROR_STOP=1 -1 -f -
```
If you can review this, that would help with my `podman` workflow too.

Reply:
I solved it by `pipx install docker-compose`, making sure I have the (working) docker-compose v1 on my path.